nov.el

```diff
@@ -797,6 +799,38 @@ Saving is only done if `nov-save-place-file' is set."
 (add-hook 'nov-mode-hook 'nov-add-to-recentf)
 (add-hook 'nov-mode-hook 'hack-dir-local-variables-non-file-buffer)
 
+
```

Comment:
Why a form feed without a comment?

Reply:
It is a ^L which used to separate different parts of code.
The ^L without comment seems will be skipped by elisp reader, so I think it OK to have no comment syntax here.